echopype/tests/echodata/test_echodata.py

```diff
@@ -311,6 +319,49 @@ def test_get_dataset(self, converted_zarr):
         assert result is None
         assert isinstance(ed_result, xr.Dataset)
 
+    @pytest.mark.parametrize("consolidated", [True, False])
+    def test_to_zarr_consolidated(self, mock_echodata, consolidated):
+        """
+        Tests to_zarr method. Currently uses a mock echodata object
+        When consolidated=True, every to_zarr call in ea. group will use the flag
+        """
+        zarr_path = Path('test.zarr')
+        mock_echodata.to_zarr(str(zarr_path), consolidated=consolidated, overwrite=True)
+
+        check = True if consolidated else False
+        zmeta_path = zarr_path / ".zmetadata"
+
+        assert zmeta_path.exists() is check
+
+        if check is True:
+            # Check that every group is in
+            # the zmetadata if consolidated
+            expected_zgroups = [
+                ".zgroup",
+                "Environment/.zgroup",
+                "Platform/.zgroup",
```

Comment:
I noticed that `"Platform/NMEA/.zgroup"` is not here. This should be added. I will also make a comment on the mock EchoData creation.